charts/ckan/templates/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.ckan.ingress.enabled -}}
```

Comment:
You should just check for `.Values.ckan.ingress.baseDomain` or in future `.Values.ckan.ingress.host`, why force the user to set 2 fields if one is enough?